File: azure-website/SiriService/StopVisit.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace SiriBackend
{
    public class StopVisit
    {
        public string LineRef { get; set; }
        public string RouteShortName { get; set; }
        public string RouteLongName { get; set; }
        public DateTime ExpectedArrivalTime { get; set; }
        public string PublishedLineName { get; set; }
        public string DestinationRef { get; set; }
    }
}

File: android-app/app/src/main/java/com/tahanot/entities/StopVisit.java
package com.tahanot.entities;

import java.util.Date;


public class StopVisit {    
	private static final long MSEC_IN_SEC = 1000;
	public String LineRef;
    public String RouteShortName;
    public String RouteLongName;
    public String ExpectedArrivalTime;
    public String PublishedLineName;
    public String DestinationRef;

    public long getExpectedInSeconds(Date responseTimestamp) {
        String s = ExpectedArrivalTime;
        long msecSince1970 = Long.parseLong(s.substring(6, s.length() - 2));
        Date expectedArrivalTime = new Date(msecSince1970);

        return (expectedArrivalTime.getTime() - responseTimestamp.getTime()) / MSEC_IN_SEC;
    };
}
